x-pack/plugin/deprecation/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/DeprecationHttpIT.java

```diff
@@ -460,6 +460,58 @@ public void testDeprecationMessagesCanBeIndexed() throws Exception {
 
     }
 
+    /**
+     * Check that a deprecation message with CRITICAL level can be recorded to an index
+     */
+    public void testDeprecationCriticalWarnMessagesCanBeIndexed() throws Exception {
```

Comment:
Now that the default is WARN, we needed to cover the case of a non-compat critical warning.